src/helpers/enrich-dataset.ts

```diff
@@ -79,10 +79,13 @@ export function enrichDataset(dataset: HubDataset, hubsite: HubSite): Feature {
 
     if (isLayer(dataset)) {
         additionalFields.accessUrlGeoJSON = downloadLinkFor('geojson');
+        additionalFields.durableUrlGeoJSON = generateDurableDownloadUrl(dataset.id, siteUrl, 'geojson');
```

Comment:
Wait, we want to even keep the old links then if we are planning on introducing these durable ones?

Reply:
For now yes. Durable links will stay experimental until we finalize the new download service which hasn't been released to production yet and these durable links will only be used for testing hub feeds on DEV and QA as required by the issue [2840]( https://devtopia.esri.com/dc/hub/issues/2840). Production hub feeds will be using old links.  

Once, download service gets released to production, these old links will be removed from the provider.